### DISCUSSION TITLE
How can I use lexical data in Flutter

### DISCUSSION THREAD

**Discussion by RevTpark** [facebook/lexical#2905] [Q&A]
I have a use case wherein we use Lexical Editor on the web app(NextJS) and store the output directly in the database. The data can then be directly shown on the web app(i.e. no need to parse the stored data as lexical does it for us). Now we also have a mobile app(Flutter) and we also want to display the stored data on the app. But the problem is that we have to parse that data and show it which is a complex task in itself. Is there a package or a better way how I can display the complex data structure in flutter? or how does the parser work in the backend so I can try to replicate the same logic?

**Comment by raghadzeidan** [comment]
We are facing the exact same problem - is an SDK lexical for flutter/dart that is in sight any time?
How did you bypass this issue? We rendered the html on mobile

**Comment by denethp** [comment]
Are there any solutions to this.
